src/st/no3/48A84.c

```diff
@@ -179,7 +179,62 @@ void func_801CC90C(Entity* arg0) {
 // stronger version of warg with jump and flame attack
 INCLUDE_ASM("st/no3/nonmatchings/48A84", EntityStrongWarg);
 
-INCLUDE_ASM("st/no3/nonmatchings/48A84", EntityUnkId30);
+void EntityUnkId30(Entity* self) {
+    Entity* entity;
+    u16 animCurFrame;
+    u16* hitboxPtr;
+
+    entity = self - 1;
+    if (self->step == 0) {
+        if (self->params != 0) {
+            InitializeEntity(&D_80180B30);
```

Comment:
```suggestion
            InitializeEntity(D_80180B30);
```
`D_80180B30` is already an array, you do not need to get the address with `&`. Same for the other `InitializeEntity` parameters

Reply:
Thanks, I originally had these not as arrays in the scratch, but then made them arrays in the .h for consistency and forgot to update this.